roles/kubernetes/node/templates/manifests/kube-vip.manifest.j2

```diff
@@ -119,6 +119,6 @@ spec:
   hostNetwork: true
   volumes:
   - hostPath:
-      path: /etc/kubernetes/admin.conf
+      path: /etc/kubernetes/{% if inventory_hostname == (groups['kube_control_plane'] | first) and ((stat_kube_vip_super_admin.stat.exists and stat_kube_vip_super_admin.stat.isreg) or (not kubeadm_already_run.stat.exists )) %}super-{% endif %}admin.conf
```

Comment:
I would rather have a variable elsewhere so it doesn't make the template hard to read. 

```yaml
# default value for all nodes
set_fact: 
  kube_vip_admin_conf:  admin.conf

# special case
set_fact:
  kube_vip_admin_conf: super-admin.conf
when:
  - inventory_hostname == groups['kube_control_plane'] | first
  - (stat_kube_vip_super_admin.stat.exists and stat_kube_vip_super_admin.stat.isreg) or ((not kubeadm_already_run.stat.exists )
```
 The template is then:
 ```
   volumes:
  - hostPath:
      path: /etc/kubernetes/{{kube_vip_admin_conf}}
```